Formula/makensis.rb

```diff
@@ -32,8 +39,13 @@ def install
     # https://sourceforge.net/p/nsis/bugs/1085/
     ENV.libstdcxx if ENV.compiler == :clang
 
+    # requires zlib (win32) to build utils
+    resource("zlib-win32").stage do
```

Comment:
It might be clearer to do something like
```ruby
(buildpath/"zlib").install resoruce("zlib-win32")
```
Then below `ZLIB_W32=#{buildpath}/zlib`.